src/main/java/Bitwise.java

```diff
@@ -0,0 +1,84 @@
+import java.util.Scanner;
+
+public class Bitwise {
+    // ASCII art from https://ascii-generator.site/t/
+    private static final String LOGO = " ____   _  _              _            \n"
+            + "|  _ \\ (_)| |            (_)           \n"
+            + "| |_) | _ | |_ __      __ _  ___   ___ \n"
+            + "|  _ < | || __|\\ \\ / /| |/ __| / _ \\ \n"
+            + "| |_) || || |_  \\ V  V / | |\\__ \\|  __/\n"
+            + "|____/ |_| \\__|  \\_/\\_/  |_||___/ \\___|\n"
+            + "                                       \n";
+    private static final String SECTION_BREAK = "==================================================\n";
+    private static final String LINE_BREAK = "--------------------------------------------------\n";
+    private static final String INDENTATION = "        ";
+    private static Task[] tasksList = new Task[100];
+    private static int numberOfTasks = 0;
+
+    public static void main(String[] args) {
+        System.out.println(SECTION_BREAK + INDENTATION + "Hello from\n" + LOGO);
+        System.out.print(INDENTATION + "How may I help you today?\n" + LINE_BREAK);
+        mainManager();
+        System.out.println(LINE_BREAK + INDENTATION + "Bye, see you soon!\n" + SECTION_BREAK);
+    }
```

Comment:
i like that your main is short and sweet because of the function being utilised from elsewhere